src/components/content/order/orderStatus/OrderSubmitResult.tsx

```diff
@@ -19,13 +21,54 @@ export const OrderSubmitResult = ({
     type,
     stopWatch,
     contactServiceDetails,
+    retryRequest,
+    deployedServiceDetails,
+    onClose,
 }: {
     msg: string | React.JSX.Element;
     uuid: string;
     type: 'success' | 'error';
     stopWatch: StopwatchResult;
     contactServiceDetails: ServiceProviderContactDetails | undefined;
+    retryRequest: () => void;
+    deployedServiceDetails: DeployedServiceDetails | undefined;
+    onClose: () => void;
 }): React.JSX.Element => {
+    const redeployFailedDeploymentQuery = useRedeployFailedDeploymentQuery();
+    stopWatch = useStopwatch({
+        autoStart: true,
+    });
+    const reDeploymentService = () => {
+        redeployFailedDeploymentQuery.mutate(uuid);
+    };
+
+    if (redeployFailedDeploymentQuery.isPending) {
```

Comment:
and for `.isPending` -> the alert must be similar to what we  have in fresh order where we show `Request submission in-progress` status